### PR TITLE
MDP-166: Footer component 

### PR DESCRIPTION
**Pull request by drerchi**
https://velocity.globallogic.com/jira/browse/MDP-166

### CODE REVIEW

File: src/components/Icon.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React, { FC } from "react";
+interface IProps {
+  src: string;
+  className?: string;
+}
+
+const Icon: FC<IProps> = props => <img src={props.src} className="w-full" />;
```
Comment (pggl) [inline src/components/Icon.tsx]: why is here w-full class?

File: src/components/footer/Contacts.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import Phone from "../../static/svg/icon-phone.svg";
+import Email from "../../static/svg/email.svg";
+import Icon from "../Icon";
+import { black } from "../../style/colors";
```
Comment (pggl) [inline src/components/footer/Contacts.tsx]: why is here black?

File: src/components/footer/Contacts.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    </div>
+    <div className="pb-1 flex justify-start items-center">
+      <div className="mr-3 w-8 lg:w-4">
+        <Icon src={Email} />
+      </div>
+      <div className="font-lato text-xs black ">
+        <a
+          href="mailto:chatkosice@gmail.com"
+          className="no-underline color-black"
+          style={{ color: black }}
```
Comment (pggl) [inline src/components/footer/Contacts.tsx]: you don't need this, black is already here in class

File: src/components/footer/index.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import React, { FC } from "react";
+import Logo from "./Logo";
+import ICO from "./Info";
+import Icon from "../Icon";
+import Contacts from "./Contacts";
+import Location from "./Location";
+import Insta from "../../static/svg/instagram.svg";
```
Comment (pggl) [inline src/components/footer/index.tsx]: why is there shorhand ? use fullname

File: src/stories/index.tsx [hunk 5]
```diff
@@ -3,5 +3,7 @@ import { storiesOf } from "@storybook/react";
 import Example from "../components/Example";
 import "../style/index.css";
 import "../style/fonts.css";
+import Footer from "../components/footer";
 
 storiesOf("Example", module).add("example", () => <Example />);
+storiesOf("Footer", module).add("responsive footer", () => <Footer />);
```
Comment (pggl) [inline src/stories/index.tsx]: add mobile version here

File: src/style/custom.css [hunk 6]
```diff
@@ -1,3 +1,53 @@
...
+    }
+    .lg\:social-icons{
+      padding-right: 40px;
+      padding-bottom: 230px;
+    }
+    .overflow-ovals{
+      position: absolute;
+      overflow: hidden;
+      width: 292.6px;
+      height: 408.7px;
```
Comment (pggl) [inline src/style/custom.css]: why is here this magic number?

**Review by pggl — CHANGES_REQUESTED** [review]

File: src/components/footer/Contacts.tsx [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+      <div className="mr-3 w-4">
+        <Icon src={Phone} />
+      </div>
+      <div className="font-lato text-xs text-black">+421 905 271 220</div>
+    </div>
+    <div className="pb-1 flex justify-start items-center">
+      <div className="mr-3 w-8 lg:w-4">
+        <Icon src={Email} />
+      </div>
+      <div className="font-lato text-xs black ">
```
Comment (pggl) [inline src/components/footer/Contacts.tsx]: do we have black classname ?

File: src/components/footer/Contacts.tsx [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+          className="no-underline text-black"
+        >
+          chatkosice@gmail.com
+        </a>
+      </div>
+    </div>
+  </div>
+);
+
+export default Contacts;
```
Comment (pggl) [inline src/components/footer/Contacts.tsx]: why is plural, are there more contacts?

File: src/components/footer/index.tsx [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+import Logo from "./Logo";
+import ICO from "./Info";
+import Icon from "../Icon";
+import Contacts from "./Contacts";
+import Location from "./Location";
+import Instagram from "../../static/svg/instagram.svg";
+import Facebook from "../../static/svg/facebook-square.svg";
+
+const Footer: FC = () => (
+  <div>
```
Comment (pggl) [inline src/components/footer/index.tsx]: why is here this empty div?

File: src/stories/index.tsx (hunk 5 above)
Comment (pggl) [inline src/stories/index.tsx]: I think "Footer" is enough for name, so remove "responsive footer"

Comment (pggl) [reply src/stories/index.tsx]: and add stories for icons

File: src/style/custom.css [hunk 10]
```diff
@@ -1,3 +1,53 @@
...
+      right: 0;
+      z-index: 1;
+      padding-right: 64px;
+      padding-bottom: 128px;
+    }
+    .lg\:social-icons{
+      padding-right: 40px;
+      padding-bottom: 230px;
+    }
+    .overflow-ovals{
```
Comment (pggl) [inline src/style/custom.css]: if we use it only on one place don't put it here and create styled component instead

**Review by pggl — CHANGES_REQUESTED** [review]

File: src/components/footer/Contacts.tsx (hunk 7 above)
Comment (drerchi) [reply src/components/footer/Contacts.tsx]: done

File: src/components/footer/index.tsx (hunk 9 above)
Comment (drerchi) [reply src/components/footer/index.tsx]: done

File: src/stories/index.tsx (hunk 5 above)
Comment (drerchi) [reply src/stories/index.tsx]: done

File: src/components/footer/Contacts.tsx (hunk 8 above)
Comment (drerchi) [reply src/components/footer/Contacts.tsx]: done

File: src/style/custom.css (hunk 10 above)
Comment (drerchi) [reply src/style/custom.css]: done

File: src/style/custom.css [hunk 11]
```diff
@@ -1,3 +1,46 @@
...
+    .lg\:social-icons{
+      padding-right: 40px;
+      padding-bottom: 230px;
+    }
+   }
+
 .link {
   @apply font-gilbert no-underline inline-block font-bold text-base text-black;
-}
+}
```
Comment (garySiladi) [inline src/style/custom.css]: no eof. add end of file newline

File: src/components/Icon.tsx [hunk 12]
```diff
@@ -0,0 +1,8 @@
+import React, { FC } from "react";
+interface IProps {
+  src: string;
+}
+
+const Icon: FC<IProps> = props => <img src={props.src} className="w-full" />;
```
Comment (garySiladi) [inline src/components/Icon.tsx]: could destructure props `const Icon: FC<IProps> = ({ src }) => ;`

Comment (drerchi) [reply src/components/Icon.tsx]: done

File: src/style/custom.css (hunk 11 above)
Comment (drerchi) [reply src/style/custom.css]: done

File: src/style/custom.css [hunk 13]
```diff
@@ -1,3 +1,47 @@
...
+      padding-right: 40px;
+      padding-bottom: 230px;
+    }
+
+   }
+
 .link {
   @apply font-gilbert no-underline inline-block font-bold text-base text-black;
-}
+  }
```
Comment (garySiladi) [inline src/style/custom.css]: este vymazes tie medzery pred zatvorkou?

Comment (drerchi) [reply src/style/custom.css]: done

**Review by garySiladi — DISMISSED** [review]
nice

**Comment by garySiladi** [comment]
contact Peter for re-review

File: src/components/Icon.tsx [hunk 14]
```diff
@@ -0,0 +1,8 @@
+import React, { FC } from "react";
+interface IProps {
```
Comment (pggl) [inline src/components/Icon.tsx:3]: add space under interface

File: src/style/custom.css [hunk 15]
```diff
@@ -1,3 +1,47 @@
+@responsive{
```
Comment (pggl) [inline src/style/custom.css:1]: we don't want to reuse this so create styled component for this

**Review by pggl — CHANGES_REQUESTED** [review]

File: src/components/Icon.tsx (hunk 14 above)
Comment (drerchi) [reply src/components/Icon.tsx:3]: done

**Review by pggl — APPROVED** [review]
good work

**Pull request merged**